mne/utils/numerics.py

```diff
@@ -663,13 +663,15 @@ def object_size(x):
     x : object
         Object to approximate the size of.
         Can be anything comprised of nested versions of:
-        {dict, list, tuple, ndarray, str, bytes, float, int, None}.
+        {dict, list, tuple, ndarray, str, bytes, float, int, None,
+        Digitization}.
```

Comment:
maybe the changes in this file are not needed as Digitization is just a list?

Reply:
great catch!